examples/go-to/index.js

```diff
@@ -0,0 +1,29 @@
+/* eslint react/prop-types: 0, react/jsx-sort-props: 0 */
+
+import React from 'react';
+
+import ReactPDF, {
+  Page,
+  Document,
+  Link,
+  View,
+  Image,
+} from '../../dist/react-pdf.es.js';
+
+const doc = (
+  <Document>
+    <Page size="A4">
+      <Link href="#myDest">Link</Link>
```

Comment:
@diegomura the `href` should actually be `src`, and should be explained in the [official document](https://react-pdf.org/components#link); it took me weeks to realize that it's aleady been implemented in the released version :)

Reply:
@y0n3t4n1 both are actually supported. What version are you using?

Reply:
I'm on 1.6.14.  I guess adding `href?` to `LinkProps` would be nice
```javascript
    interface LinkProps extends NodeProps {
      /**
       * Enable/disable page wrapping for element.
       * @see https://react-pdf.org/components#page-wrapping
       */
      wrap?: boolean;
      debug?: boolean;
      src: string;
      children?: React.ReactNode;
    }
```

Reply:
Well yes, thanks. I'll add it! Not sure I understood what tooks you a week thought since `src` is also valid